Sources/EdgeNetworkHandlers/RequestBuilder.swift

```diff
@@ -47,10 +47,12 @@ class RequestBuilder {
     }
 
     /// Builds the request payload with all the provided parameters and events.
-    /// - Parameter events: List of `Event` objects. Each event is expected to contain a serialized `ExperienceEvent`
+    /// - Parameters:
+    ///   - events:  List of `Event` objects. Each event is expected to contain a serialized `ExperienceEvent`.
+    ///   - config: Edge configuration to extract the datastream Identifier from.
     /// encoded in the `Event.data` property.
     /// - Returns: A `EdgeRequest` object or nil if the events list is empty
-    func getPayloadWithExperienceEvents(_ events: [Event]) -> EdgeRequest? {
+    func getPayloadWithExperienceEvents(events: [Event], config: [String: String]) -> EdgeRequest? {
```

Comment:
we shouldn't need the config as part of the experience event, all config settings would at top level metadata and/or the configId query param

Reply:
It should be available in the requestMetadata on L62 right? 
https://github.com/adobe/aepsdk-edge-ios/pull/398/files/751a5af8429aed0a339b2a08f7f69ef51aa0710b#diff-4be59ca881aa157bb599eab4779d6cc7c0c10f9bb1d0187d8759ec6e1c636fb2R62